### ISSUE TITLE
Feature Request - Add more than one point

### ISSUE THREAD

**Issue by akmzero** [meshtastic/meshtastic_linkplanner#10]
Would it be possible/plausible to enable being able to plot more than one location at a time?

**Comment by mrpatrick1991** [comment]
Yes, and this is a good idea because we could account for multi-hops. Would require front-end work.

Open to PR's, I have limited time over the next couple weeks to try this.
